README.md

```diff
@@ -45,99 +44,25 @@ sudo dnf install cmake libXrandr-devel libXinerama-devel libXcursor-devel
 
 # Ecosystem
 
-`Makie.jl` is the metapackage for a rich ecosystem, which consists of [`GLMakie.jl`](https://github.com/JuliaPlots/GLMakie.jl), [`CairoMakie.jl`](https://github.com/JuliaPlots/CairoMakie.jl) and [`WGLMakie.jl`](https://github.com/JuliaPlots/WGLMakie.jl) (the backends); [`AbstractPlotting.jl`](https://github.com/JuliaPlots/AbstractPlotting.jl) (the bulk of the package); and [`StatsMakie.jl`](https://github.com/JuliaPlots/StatsMakie.jl) (statistical plotting support, as in [`StatsPlots.jl`](https://github.com/JuliaPlots/StatsPlots.jl)).
+Makie is the metapackage for a rich ecosystem, which consists of [GLMakie.jl](https://github.com/JuliaPlots/GLMakie.jl), [CairoMakie.jl](https://github.com/JuliaPlots/CairoMakie.jl) and [WGLMakie.jl](https://github.com/JuliaPlots/WGLMakie.jl) (the backends); [AbstractPlotting.jl](https://github.com/JuliaPlots/AbstractPlotting.jl) (the bulk of the package); and [StatsMakie.jl](https://github.com/JuliaPlots/StatsMakie.jl) (statistical plotting support, as in [StatsPlots.jl](https://github.com/JuliaPlots/StatsPlots.jl)
```

Comment:
Removed `).` to close the parenthesis and the punctuation here. (Pretty sure it's a typo)